clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1868,11 +1882,23 @@ private void sendPushJobDetailsToController() {
     updatePushJobDetailsWithConfigs();
     updatePushJobDetailsWithLivenessHeartbeatException();
 
+    /**
+     *  This logic is added to migrate push job details system store from Aggregate mode to Active/Active mode.
+     *  If it is multi-region, it will dual write push job details to parent RT and child region RT.
+     *  If it is single-region, it will only write push job details to child region RT.
```

Comment:
Why can't parent controller write it to child colo's RT instead? Why do we need to make a controller client to an arbitrary child colo?

Reply:
There is also stuff in parent controller that emits metrics about push job success/failures that we'll need to handle

Reply:
Yeah you are right...I think I read the code wrong. It probably cannot work alone within VPJ changes. I will probably just change to dual write in parent controller.